frontends/web/src/locales/de/app.json

```diff
@@ -483,23 +480,21 @@
         },
         "privacyPolicy": "MoonPay-Datenschutzrichtlinie",
         "protection": {
-          "description": "Die BitBoxApp sammelt beim Kauf von {{name}} keine Daten. Die eingehenden Transaktionen werden wie reguläre Transaktion behandelt. MoonPay muss einige persönliche Daten sammeln, um funktionieren zu können. In deren Datenschutzbestimmungen wird ausführlich erläutert, wie mit diesen Daten umgegangen wird.",
-          "descriptionGeneric": "Die BitBoxApp erhebt beim Kauf von {{name}} keinerlei Daten, die eingehenden Gelder werden wie eine normale Transaktion behandelt. Partnerbörsen müssen jedoch einige Informationen sammeln, um zu funktionieren. Einzelheiten zum Umgang mit Daten kannst du der jeweiligen Datenschutzerklärung entnehmen.",
+          "descriptionGeneric_bitcoin": "Die BitBoxApp erhebt beim Kauf von Bitcoin keinerlei Daten, die eingehenden Gelder werden wie eine normale Transaktion behandelt. Partnerbörsen müssen jedoch einige Informationen sammeln, um zu funktionieren. Einzelheiten zum Umgang mit Daten kannst du der jeweiligen Datenschutzerklärung entnehmen.",
```

Comment:
hmmmmmm.... I think I accidentally nuked this and pasted an older verion. cc @sutterseba do you know which version is newer?